gateway-js/src/index.ts

```diff
@@ -203,7 +204,7 @@ export class ApolloGateway implements GraphQLService {
   // Configure the endpoint by which gateway will access its precomposed schema.
   // For now, `null` is default and means to continue using the legacy managed mode.
```

Comment:
I think this comment that suggests that `null` is the default is the only thing that seems off to me.

Reply:
Good catch!